src/pages/documentation/media-for-edge-network/api-reference.md

```diff
@@ -0,0 +1,462 @@
+
+Android
+
+<Tabs query="platform=android&api=track-play"/>
+
+iOS
+
+<Tabs query="platform=ios&api=track-play"/>
+
+### trackPause
+
+Tracks the media pause.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
+
+Android
+
+<Tabs query="platform=android&api=track-pause"/>
+
+iOS
+
+<Tabs query="platform=ios&api=track-pause"/>
+
+### trackComplete
+
+Tracks the completion of the media playback session. Call this method only when the media has been completely viewed. If the viewing session is ended before the media is completely viewed, use [`trackSessionEnd`](#trackSessionEnd) instead.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
+
+Android
+
+<Tabs query="platform=android&api=track-complete"/>
+
+iOS
+
+<Tabs query="platform=ios&api=track-complete"/>
+
+### trackSessionEnd
+
+Tracks the end of a media playback session. Call this method when the viewing session ends, even if the user has not viewed the media to completion. If the media is viewed to completion, use [`trackComplete`](#trackComplete) instead.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
+
+Android
+
+<Tabs query="platform=android&api=track-session-end"/>
+
+iOS
+
+<Tabs query="platform=ios&api=track-session-end"/>
+
+### trackError
+
+Tracks an error in media playback.
+
+| Parameter | Description | Required |
+| :--- | :--- | :---: |
+| `errorId` |  The custom error Identifier. | Yes |
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
+
+Android
+
+<Tabs query="platform=android&api=track-error"/>
+
+iOS
+
+<Tabs query="platform=ios&api=track-error"/>
+
+### trackEvent
+
+Tracks media events.
+
+| Parameter | Description | Required |
+| :--- | :--- | :--: |
+| `event` | The media event being tracked, use [Media event constants](#media-events-constants) | Yes|
+| `info` | For an `AdBreakStart` event, the AdBreak information is created by using the [`createAdBreakObjectWith`](#createAdBreakObjectWith) method.<br/> For an `AdStart` event, the Ad information is created by using the [`createAdObjectWith`](#createAdObjectWith) method.<br/> For a `ChapterStart` event, the Chapter information is created by using the [`createChapterObjectWith`](#createChapterObjectWith) method.<br/> For a `StateStart` and `StateEnd` event, the State information is created by using the [`createStateObjectWith`](#createStateObjectWith) method. | Yes/No* |
+| metadata | Optional context data can be provided for `AdStart` and `ChapterStart` events. This is not required for other events. | No |
+
+<InlineAlert variant="info" slots="text"/>
+
+*`info` is a required parameter for `AdBreakStart`, `AdStart`, `ChapterStart`, `StateStart`, `StateEnd` events. Not set for any other event types.
+
+<InlineAlert variant="info" slots="text"/>
+
+Ad ping interval can now be customized to duration between 1 to 10 seconds using [`createTrackerWithConfig`](#createTrackerWithConfig) API.
```

Comment:
Can you double check if anchor links are case sensitive?  I am not able to go to them with upper case letter in the links.  e.g (#createTrackerWithConfig), but it works with (#createtrackerwithconfig).